altair/vegalite/v5/api.py

```diff
@@ -1438,7 +1437,9 @@ def selection(type: Optional[SelectionType_T] = Undefined, **kwds: Any) -> Param
 
 def selection_interval(
     name: str | None = None,
-    value: Optional[Any] = Undefined,
+    value: Optional[
+        PrimitiveValue_T | Temporal | SchemaBase | Sequence[SchemaBase | Map] | Map
+    ] = Undefined,
```

Comment:
### TODO
- Implement the plan from #3656

Reply:
Resolved in [`30aa4ba` (#3653)](https://github.com/vega/altair/pull/3653/commits/30aa4baa537308ffa7e27effb2bcd386ee9a701e)